localisation/00_zuwc_united_localisation.csv

```diff
@@ -446,7 +446,7 @@ EVTDESC_WCFEL_1005;The world of Azeroth is threatened once again. Demons of the
 EVTOPTA_WCFEL_1005_case_01;Azeroth will fall!;Azeroth va tomber�!;Azeroth wird fallen!;;;;;;;;;;;x
 EVTOPTA_WCFEL_1005_case_02;The Burning Legion must be stopped!;La L�gion ardente doit �tre arr�t�e�!;Die brennende Legion muss gestoppt werden!;;;;;;;;;;;x
 EVTTITLE_WCCLS_1010;The Defeat of the Burning Legion;La D�faite de la L�gion ardente;Die Niederlage der brennenden Legion;;;;;;;;;;;x
-EVTDESC_WCFEL_1010;The armies of the Burning Legion have been defeated! Only a handful of demons attempts to continue the war effort in Azeroth, but they are nothing compared to our superior armies. The leader of the invasion, [Root.GetTitledFirstName] has been banished, and it is unlikely that he will lead a new invasion. Thankfully, it is unlikely that the armies of demons of the Burning Legion will attempt to take over our world again in the near future.;;Die Armeen der brennenden Legion wurden besiegt! Nur eine Handvoll D�monen versuchen, den Krieg gegen Azeroth fortzusetzen, aber sie stellen keine Gefahr mehr dar. Der Anf�hrer dieser Invasion, [Root.GetTitledFirstName], wurde verbannt und es ist unwahrscheinlich, dass er eine weitere Invasion anf�hren wird. Zweifellos werden die D�monenhorden der brennenden Legion erneut versuchen, diese Welt zu erobern, aber nicht in der n�chsten Zeit.;;;;;;;;;;;x
+EVTDESC_WCFEL_1010;The armies of the Burning Legion have been defeated! Only a handful of demons attempts to continue the war effort in Azeroth, but they are nothing compared to the superior armies defending this world. The leader of the invasion, [Root.GetTitledFirstName] has been banished, and it is unlikely that he will lead a new invasion. Thankfully, it is unlikely that the armies of demons of the Burning Legion will attempt to take over our world again in the near future.;;Die Armeen der brennenden Legion wurden besiegt! Nur eine Handvoll D�monen versuchen, den Krieg gegen Azeroth fortzusetzen, aber sie stellen keine Gefahr mehr dar. Der Anf�hrer dieser Invasion, [Root.GetTitledFirstName], wurde verbannt und es ist unwahrscheinlich, dass er eine weitere Invasion anf�hren wird. Zweifellos werden die D�monenhorden der brennenden Legion erneut versuchen, diese Welt zu erobern, aber nicht in der n�chsten Zeit.;;;;;;;;;;;x
```

Comment:
Changed attempts to attempting (due to tense problems) and unlikely to implausible (because unlikely had been used only a few words before.
```suggestion
EVTDESC_WCFEL_1010;The armies of the Burning Legion have been defeated! Only a handful of demons are attempting to continue the war effort in Azeroth, but they are nothing compared to the superior armies defending this world. The leader of the invasion, [Root.GetTitledFirstName] has been banished, and it is unlikely that he will lead a new invasion. Thankfully, it seems implausible that the armies of demons of the Burning Legion will attempt to take over our world again in the near future.;;Die Armeen der brennenden Legion wurden besiegt! Nur eine Handvoll D�monen versuchen, den Krieg gegen Azeroth fortzusetzen, aber sie stellen keine Gefahr mehr dar. Der Anf�hrer dieser Invasion, [Root.GetTitledFirstName], wurde verbannt und es ist unwahrscheinlich, dass er eine weitere Invasion anf�hren wird. Zweifellos werden die D�monenhorden der brennenden Legion erneut versuchen, diese Welt zu erobern, aber nicht in der n�chsten Zeit.;;;;;;;;;;;x
```

Reply:
Oh, totally missed that. Fixed